### DISCUSSION TITLE
Integration of CivitAI

### DISCUSSION THREAD

**Discussion by saksh-aam** [civitai/civitai#555] [General]
Is there a way that exist to integrate the CivitAI models to my application, wherein I can run these models to generate the images?
(Prefereable way would be without downloading the models as .safetensors files.)

**Comment by yellow2fatty** [comment]
AFAIK, civit ai is a community focusing on sharing. I know a startup providing API for inferencing existing models listed on civitai. Do you want me to share their info?

**Reply by saksh-aam** [reply]
Yes, that would help.
But how can I build the same API support? Can you share some info on that?

**Reply by yellow2fatty** [reply]
> Yes, that would help. But how can I build the same API support? Can you share some info on that?

This is the [one ](https://www.genartai.site/)I am talking about. I am not sure what you want to builld with the API.
